synapse/telepath.py

```diff
@@ -126,6 +126,10 @@ async def getAhaProxy(urlinfo):
         mesg = f'getAhaProxy urlinfo has no host: {urlinfo}'
         raise s_exc.NoSuchName(mesg=mesg)
 
+    if not aha_clients:
+        mesg = f'No aha clients are available to lookup {host}'
+        raise s_exc.NotReady(mesg=mesg)
```

Comment:
Refocus this message to be user-perspective rather than developer perspective :D Something like `No aha servers registered.` etc etc